engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-action-packages.controller.js

```diff
@@ -84,5 +84,22 @@ angular.module('Bastion.content-hosts').controller('ContentHostsBulkActionPackag
             return deferred.promise;
         };
 
+        $scope.performViaRemoteExecution = function(action, customize) {
+            var remoteAction, selectedHosts = $scope.nutupane.getAllSelectedResults();
+            if ($scope.content.contentType == "package_group") {
+                remoteAction = "group_" + action;
+            } else if ($scope.content.contentType == "package") {
+                remoteAction = "package_" + action;
+            }
+            form = $('#packageActionForm');
+            form.attr('action', '/katello/remote_execution');
+            form.attr('method', 'post');
+            form.find('input[name=remote_action]').val(remoteAction);
+            form.find('input[name=authenticity_token]').val(AUTH_TOKEN.replace(/&quot;/g,''));
+            form.find('input[name=customize]').val(customize);
+            form.find('input[name=content_host_ids]').val(selectedHosts.included.ids.join(','));
+            form.find('input[name=scoped_search]').val(selectedHosts.included.search);
+            form.submit();
```

Comment:
I'm not saying I'm particularly happy about this approach, but my other tries ended up miserably :/ I also don't have resources to rewrite the whole remote execution interface in Angular and don't think it would move us forward (we would still be a step behind the remote execution UI interface). Trying to do it the angular way just seemed over-complicated, and not worth, since the goal here really is get into the non-angular part of Remote Execution. The last thing that made be use the jQuery was not ability to generate the `get` query from params in pure angular, so I needed to use `$.params` instead, which just made me think that in this particular case, this is the simplest solution.

I'm open to other approaches, but to be honest, I've spent a lot of time trying to go the Angular way, but it just didn't feel right. And if I were you right now seeing the code, I would suggest the same as you do right now:) But with my history of trying, I'm out of other nicer ideas and this brings me just what I need. I realize this is not what Angular people do, but they don't have a hybrid UI to deal with, as we do. 

What I had before was:
1. in the `customize first` case, I needed to construct a get request to `katello/remote_execution/new` page, and use `$window.location.href` to get there
2. in the `run immediately` case, I needed to define a resource in angular, to trigger an API that would return a job invocation id, just to set the `$window.location.href` to `/job_invocations/:id`. And if there was an error while preparing the job invocation, I would need to format that error messages as well.

When going the `jQuery` way, I've solved both cases with one code and, in case of `run immediately` case, I get to the job invocation form with validation errors for free. 

I hope that explains more my intentions and reasons why ending up in this.